Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -965,9 +964,16 @@ void WebsiteDataStore::removeData(OptionSet<WebsiteDataType> dataTypes, const Ve
     }
 }
 
-Ref<DeviceIdHashSaltStorage> WebsiteDataStore::protectedDeviceIdHashSaltStorage()
+Ref<DeviceIdHashSaltStorage> WebsiteDataStore::ensureDeviceIdHashSaltStorage()
```

Comment:
Per our usual style: `ensureDeviceIdHashSaltStorage()` should return a `DeviceIdHashSaltStorage&`. A `ensureProtectedDeviceIdHashSaltStorage()` overload (which returns a `Ref<>`) may be added for call sites that need it.